xfields/beam_elements/beambeam_src/beambeam3d.h

```diff
@@ -70,6 +81,21 @@ void synchrobeam_kick(
             &dS_Sig_11_hat_star, &dS_Sig_33_hat_star,
             &dS_costheta, &dS_sintheta);
 
+    //printf("[turn: %d] Sig_11_hat_star: %.6e, Sig_33_hat_star: %.6e, dS_Sig_11_hat_star: %6e, dS_Sig_33_hat_star: %6e\n", turn, Sig_11_hat_star, Sig_33_hat_star, dS_Sig_11_hat_star, dS_Sig_33_hat_star);
+    // apply kick only if Sig_11_hat_star > 0 and Sig_33_hat_star > 0 (corresponds to num_macroparts_in_slice > 2)
```

Comment:
I would change the comment to 
```
// apply kick only if Sig_11_hat_star > 0 and Sig_33_hat_star > 0 (corresponds to sufficient macrparicle population)
```

Even if with two particles you can compute a number, computing a beam position and  beam size with two particles makes absolutely no sense

Reply:
changed comment